dashboard/config/libraries/NativeSpriteLab.interpreted.js

```diff
@@ -0,0 +1,171 @@
+function draw() {
+  executeDrawLoopAndCallbacks();
+}
+
+/* Legacy code only. Do not add any new code below here */
```

Comment:
If this is legacy code, do we have a plan to remove it?

Reply:
No :(
Unfortunately, I think the plan is that we need to support these blocks forever

Reply:
Meh. Ok. Thanks for the super clear comment so people don't add more.